ipynb/FilterAndPickOTUs.ipynb

```diff
@@ -0,0 +1,329 @@
+ "nbformat": 3,
+ "nbformat_minor": 0,
+ "worksheets": [
+  {
+   "cells": [
+    {
+     "cell_type": "heading",
+     "level": 3,
+     "metadata": {},
+     "source": [
+      "Filter and pick OTUs"
+     ]
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "* License: BSD\n",
+      "* Copyright: American Gut Project, 2014"
+     ]
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "Early in the American Gut Project, it was observed that some organisms bloomed likely as a result of increased shipping time and delay between when samples were collected and when they were put on ice (more detail can be found [here](http://americangut.org/?page_id=277)). The purpose of this notebook is to apply the filter developed in order to bioinformatically subtract these observed bloom sequences from fecal samples. It is important to apply this filter when combining data with the American Gut as to remove a potential study-effect bias as all fecal data in the American Gut has had this filter applied. The specific steps covered are:\n",
+      "\n",
+      "* Filter demultiplexed sequence data to only fecal samples\n",
+      "* Determine what sequences in the fecal samples recruit to the observed bloom sequences\n",
+      "* Remove the recruited bloom sequences from the demultiplexed sequence data\n",
+      "* Trim the filtered demultiplexed data (to reduce study bias when combining with short reads, such as the data in [Yatsunenko et al 2012](http://www.nature.com/nature/journal/v486/n7402/abs/nature11053.html))\n",
+      "* Pick OTUs against Greengenes 13_8 for the trimmed and full length filtered demultiplexed data\n",
+      "\n",
+      "The notebook is designed to support bloom filtering multiple datasets. \n",
+      "\n",
+      "The filtering is only intended to be applied to fecal data. As such, this notebook allows you to describe what metadata column and value to use so that only fecal samples are used."
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "%run cluster_utils.ipy\n",
+      "import os\n",
+      "from functools import partial\n",
+      "\n",
+      "current_home = os.environ['HOME']\n",
+      "basedir = os.path.join(current_home, \"ICU\")\n",
+      "working_dir = os.path.join(basedir, 'blooming')\n",
+      "prj_name = 'blooming'\n",
+      "trim_length = 100\n",
+      "\n",
+      "# define a submit function against qsub\n",
+      "def submit(cmd_prefix, prj_name, queue, extra_args, cmd):\n",
+      "    \"\"\"Submission wrapper to submit to qsub\"\"\"\n",
+      "    cmd = \"%s; %s\" % (cmd_prefix, cmd)\n",
+      "    return submit_qsub(cmd, job_name=prj_name, queue=queue,\n",
+      "                       extra_args=extra_args)\n",
+      "\n",
+      "# define any environment specific details for any running compute job\n",
+      "cmd_prefix = 'source ~/.bash_profile; workon ag'\n",
```

Comment:
comment indicates it is to define any env specific details... would you like that expanded further?